openshift_tools/monitoring/zagg_sender.py

```diff
@@ -151,9 +147,13 @@ def add_zabbix_keys(self, zabbix_keys, host=None, synthetic=False):
 
         self.unique_metrics += zabbix_metrics
 
+    def add_metric(self, metric, host=None, synthetic=False):
+        """ Wrapper for _add_zubbix_keys """
+        self._add_zabbix_keys(metric, host, synthetic)
+
     # Allow for 6 arguments (including 'self')
     # pylint: disable=too-many-arguments
-    def add_zabbix_dynamic_item(self, discovery_key, macro_string, macro_array, host=None, synthetic=False):
+    def _add_zabbix_dynamic_item(self, discovery_key, macro_string, macro_array, host=None, synthetic=False):
```

Comment:
Is there a reason not to just rename this to `add_dynamic_metric` directly instead of doing the forward method?